src/components/Calculator.css

```diff
@@ -0,0 +1,64 @@
+.calculator__container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 0.125em;
+  width: max-content;
+  background: #d3d3d3;
+}
```

Comment:
[OPTIONAL] 
- [ ] Your app is not quite responsive on some screen dimensions, kindly fix that to allow ease of use on more devices.

Reply:
This has bee done thank you